packages/@mantine/core/src/components/InlineInput/InlineInput.tsx

```diff
@@ -36,6 +36,8 @@ export interface InlineInputProps
   error: React.ReactNode;
   size: MantineSize | (string & {}) | undefined;
   labelPosition?: 'left' | 'right';
+  bodyElement?: any;
+  labelElement?: any;
```

Comment:
```suggestion
  bodyElement?: string;
  labelElement?: string;
```
should also be changed from any